<?xml version="1.0"?>
<launch>
    <!-- namespace -->
	<arg name="data_ns" default="/kuam/data" />

    <!-- payload_cmd_node  -->
	<arg name="process_freq" default="30.0" />

    <!-- <group ns="/kuam/maneuver"> -->
        <node pkg="kuam_payload_cmd" type="payload_cmd_node" name="payload_cmd" output="screen">
            <param name="process_freq" value="$(arg process_freq)" />
            <param name="data_ns" value="$(arg data_ns)" />
        </node>
    <!-- </group> -->
</launch>